forecasttools/idata_to_tidy.py

```diff
@@ -0,0 +1,71 @@
+"""
+Contains functions for interfacing between
+the tidy-verse and arviz, which includes
+the conversion of idata objects (and hence
+their groups) in tidy-usable objects.
+"""
+
+import re
+
+import arviz as az
+import polars as pl
+
+
+def convert_idata_forecast_to_tidydraws(
+    idata: az.InferenceData,
+    groups: list[str]
+) -> dict[str, pl.DataFrame]:
+    """
+    Creates a dictionary of polars dataframes
+    from the groups of an arviz InferenceData
+    object that when converted to a csv(s)
+    and read in R is tidy-usable.
```

Comment:
No need to mention csv. It could be saved in any format that supports tabular data. Instead of tidy-usable, refer to the tidybayes API https://mjskay.github.io/tidybayes/articles/tidybayes.html